docs/source/usage/likelihoodComputation.rst

```diff
@@ -148,59 +177,61 @@ In the case of a name collision between an output file and an existing
 file, the existing file will be *overwritten* unless the ``skip_exist``
 parameter is set *and* the complete set of output files are present.
 
-For the following examples, assume ``folder_output`` is set to ``FOLDER_OUTPUT``.
-``N`` is the template number (the value of ``i_template``), NOT zero-padded.
+For the following examples, assume ``folder_output`` is set to
+``OUT``. ``N`` is the template number (the
+value of ``i_template``), NOT zero-padded.
```

Comment:
I changed the example value of the `folder_output` parameter to `OUT` instead of `FOLDER_OUTPUT` to make the examples (below) shorter.